agent/discovery/host.go

```diff
@@ -90,3 +89,35 @@ func getHostIpAddresses() ([]string, error) {
 
 	return ipAddrList, nil
 }
+
+func GetOSVersion() string {
+	infoStat, err := host.Info()
+	if err != nil {
+		log.Errorf("Error while getting host info: %s", err)
+	}
+	return infoStat.PlatformVersion
```

Comment:
What if we get an error from `host.Info()`? We'll still have `PlatformVersion` inside `infoStat`?

Reply:
It will get its null value, in this case an empty string. I guess :sweat_smile: